### DISCUSSION TITLE
When do Nuxt plan to launch the first stable version of nuxtjs3

### DISCUSSION THREAD

**Discussion by lukangkang** [nuxt/nuxt#17938] [Questions]
I've been waiting for a whole year...

**Comment by Azema4ka** [comment]
> Я ждал целый год...

![image](https://user-images.githubusercontent.com/50888444/153876350-290c5a2f-c004-4822-bbb8-3ab686476656.png)

**Reply by lukangkang** [reply]
where you got this image? coule you please give me the original link? thanks

**Reply by Azema4ka** [reply]
> откуда у тебя это изображение? coule вы, пожалуйста, дайте мне оригинальную ссылку? Благодарность

https://twitter.com/Atinux/status/1492134698629451782?s=20&t=ZryjEGxfopiwXK6Tyr23sA

**Comment by TheAlexLichter** [comment]
Please follow https://github.com/nuxt/framework/discussions/2198 along.
